src/utils/query-params.ts

```diff
@@ -0,0 +1,30 @@
+import { areArraysEqual } from './array';
+
+export enum ValueType {
```

Comment:
nit: `ValueType` should be renamed to something more specific e.g. `QueryParamValueType`